controllers/tweet.js

```diff
@@ -1,22 +1,28 @@
 const pool = require("../db");
-const jwt = require('jsonwebtoken');
+const { getCommentsByTweet } = require("./comment");
 
 
 const getTweets = async () => {
     const query = 'SELECT * FROM tweet WHERE delete_at is NULL';
     const res = await pool.query(query);
+    const commentsByTweet = await getCommentsByTweet(res.rows[0].id);
+    res.rows[0].comments = commentsByTweet;
     return res.rows;
 };
 
 const getTweet = async (id) => {
     const query = 'SELECT * FROM tweet WHERE id=$1 AND delete_at is NULL';
     const res = await pool.query(query, [id]);
+    const commentsByTweet = await getCommentsByTweet(id);
+    res.rows[0].comments = commentsByTweet;
     return res.rows;
 };
 
 const getTweetByUser = async (id) => {
```

Comment:
```suggestion
const getTweetsByUser = async (id) => {
```
this should be plural because the relation user - tweet is 1:n, i.e. there are multiple tweets per each user

in this case you have the same problem you got in getTweets